packages/contracts/contracts/newsroom/EventStorage.sol

```diff
@@ -0,0 +1,12 @@
+pragma solidity ^0.4.19;
+
+contract EventStorage {
```

Comment:
I thought this functionality would be part of the Newsroom contract. The editor would enter the articleId, revisonId, and the string.

Reply:
EventStorage was discussed as a temporary solution. This is done on purpose to avoid permanent hacks in the Newsroom contract, as well as to avoid bloat in the Newsroom contract and face problems similar to CivilTCR.

To make it into one transaction, one could create a quick smart-contract wrapper which archives as well as publishes a new revision in one transaction